### PR TITLE
Small improvements for SDK newbies

### PR DESCRIPTION
**Pull request by nstielau**
These are a couple things I stumbled upon during my first SDK build:
* How to verify you are in the sdk chroot
* How to exit the sdk chroot
* How to enable KVM support in BIOS to allow booting

cc @bgilbert @zbwright

### CODE REVIEW

**Comment by nstielau** [comment]
I'd also like instructions on the smallest thing that can be changed, even tweaking os-release or just adding a /NICKWASHERE.txt file and then building and booting the image.

**Comment by zbwright** [comment]
looks fine to me - get @bgilbert to approve and merge away!

File: os/sdk-modifying-coreos.md [hunk 1]
```diff
@@ -75,9 +75,17 @@ cork enter
...
 
+Verify you are in the SDK chroot:
+
+```
+$ grep NAME /etc/os-release
+NAME="Container Linux by CoreOS"
+```
+
+To leave the SDK chroot, simply run `exit`.
+
```
Comment (bgilbert) [inline os/sdk-modifying-coreos.md]: This addition should be above the `To use the SDK chroot in the future` line.

File: os/sdk-modifying-coreos.md [hunk 2]
```diff
@@ -157,6 +165,8 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
+If you encounter errors with KVM, please ensure it is supported by your CPU and enabled in the BIOS.
```
Comment (bgilbert) [inline os/sdk-modifying-coreos.md]: `please` is arguably contrary to the style guide.  :disappointed:  Maybe just `verify it is supported`?

File: os/sdk-modifying-coreos.md (hunk 1 above)
Comment (bgilbert) [reply os/sdk-modifying-coreos.md]: I meant the entire addition, not just the one line.

**Pull request merged**